### ISSUE TITLE
The threshold for “vertical swipe to close” is too large

### ISSUE THREAD

**Issue by simevidas** [dimsemenov/PhotoSwipe#858]
Both Facebook and WhatsApp (the native apps) use “vertical swipe to close” when viewing images and videos in overlay, but they only require a small swipe to perform this action, which enables flicking (performing a quick swipe on a relatively small portion of the screen).

PhotoSwipe, on the other hand, does not support flicking, as the “close” action only registers if the user swipes the image all the way up/down over the edge of the screen. I think flicking is desirable, and the vertical swipe threshold should be reduced to support it.

**Comment by dimsemenov** [comment]
I agree. Will be done. Btw there is an option that controls threshold `verticalDragRange: 0.6`.
